text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,536 @@
+channels increase, this scan prolongs the process of adding a new orderer.  
+    
+## Operational problems
+There are several operational problems that affect large scale deployment and hinder efficient management of cloud 
+environments using automatic tooling.
+     
+* Channel creation (using the system channel) utilizes information contained in the system channel - e.g. MSP information 
+of the organization - in order to apply it to the newly created channel. This creates the convoluted situation in 
+which an organization that wants to update its details in the consortium must coordinate that action with third parties 
+that have write privileges to the system channel.
+
+* There is no good way to list all the channels that an orderer is a member of. 
+
+* There is no good way to dispose of the resources associated with a channel, once an orderer is no longer a member of 
+the consenters set of a that given channel.
+
+* In many cloud environments, it is necessary to restart an orderer in order for it to discover immediately that it has 
+been added to a channel it was not a part of previously, because otherwise detection takes minutes. 
+
+   
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+We introduce a new mode of channel management that does not depend on the existence of a system channel.
+
+We introduce the ability to start an OSN process without any channels. The OSN exposes a new **"channel participation" 
+API** that allows the operator to join and leave channels on that specific OSN.   
+
+A **"local orderer channel participation administrator"** is a role that is associated with the operation and 
+administration of an OSN. The identity associated with this role is able to invoke a new **"channel participation" 
+API**, and perform the following commands:
+
+- **Join** this OSN to a new or an existing channel,
+- **Remove** this OSN from an existing channel,
+- **List** the channels this OSN is a member of.   
+
+The new mode of channel participation management is meant to be incompatible with channel creation using a system 
+channel, in the sense that if the system channel exists, the `Join` operation will not be supported. 
+However, there is a transition path from a network operated with a system channel to a network that is operated without
+one: removing the system channel from all OSNs.  
+
+It should be possible to start OSN processes without channels, create a system channel with the
+channel participation API, and continue to operate the network in the usual way - creating channels through the system 
+channel. 
+
+"Mixed mode" management, that is, using the system channel to create channels on some OSNs and the 
+participation API to create channels on other OSNs is not supported and highly discouraged.  
+
+## Channel participation commands 
+In this section channel participation commands are described in a high level. The commands are presented in REST style
```

Comment:
Any specific reason to use REST rather than the existing gRPC style?

Reply:
We want to make it easier for cloud operators to manage the OSN channel participation. This way they won't need additional tools that implement gRPC, and can use a browser, for example.

Reply:
I'm a bit torn on this one ... I tend to think we should probably stick with gRPC here.  But I will not hold things up ... the APIs generally look ok and it's not hard to translate these to protobuf defs.